### PR TITLE
set image type as a class

### PR DESCRIPTION
**Pull request by axyz**
@sylvesteraswin @ytanruengsri 

Ignore compiled stuff (should be addet to gitignore?)

I'm setting the image type (when provided) as an additional class in order to apply some hack to style NON_MODEL images

I also tried the "clean" way using the custom renderItem, but that function internally saves the ref of the new image in the component instance, it is not possible to do so from outside and without that ref prev and next will not work.

### CODE REVIEW

File: build/react-gallery-swiper.js [hunk 1]
```diff
@@ -187,7 +187,7 @@ return /******/ (function(modules) { // webpackBootstrap
 	            var newImages = nextProps.images,
 	                newLazyload = nextProps.lazyLoad;
 
-	            if (images !== newImages) {
+	            if (images !== newImages || images.length !== newImages.length) {
```
Comment (avivshaked) [inline build/react-gallery-swiper.js]: is there a use case for this? Are you changing the original list somehow instead of providing a new list?

Comment (axyz) [inline build/react-gallery-swiper.js:190]: I didn't add that code, it is on the compiled code. My guess is that the previous version did not update the compiled code and this is something present in previous changes.

File: lib/GallerySwiper.js [hunk 2]
```diff
@@ -89,7 +89,7 @@ var GallerySwiper = function (_Component) {
                 newLazyload = nextProps.lazyLoad;
 
 
-            if (images !== newImages) {
+            if (images !== newImages || images.length !== newImages.length) {
```
Comment (avivshaked) [inline lib/GallerySwiper.js:92]: is there a use case for this? Are you changing the original list somehow instead of providing a new list?

Comment (axyz) [reply lib/GallerySwiper.js:92]: same

**Comment by ytanruengsri** [comment]
@axyz should I merge?

**Comment by axyz** [comment]
@ytanruengsri yep, need o merge and release to make pdp branch work

File: lib/GallerySwiper.js (hunk 2 above)
Comment (avivshaked) [reply lib/GallerySwiper.js:92]: Yeah, every time I write this comment I look up and see it's generated code... I keep going down to find the actual code :)

**Comment by avivshaked** [comment]
👍

**Comment by avivshaked** [comment]
BTW, should be minor version change, no?

**Comment by axyz** [comment]
@avivshaked done

**Pull request merged**